src/content/api/notifications.mdoc

```diff
@@ -0,0 +1,44 @@
+---
+title: Notifications
+description: Notification models
+nav:
+  path: Notifications
+  order: 1
+---
+A Notification is sent to subscribers when specific events occur in our system. Notifications may contain details about the event, and relevant metadata.
+
+## Notification Model
+
+{% properties %}
+  {% property name="id" type="string" %}
+    Notification's unique identifier.
+  {% /property %}
+
+  {% property name="type" type="string" %}
+    The event that occurred that triggered the Notification. See below for the list of types.
+  {% /property %}
+
+  {% property name="data" type="string" %}
+    The associated metadata.
+  {% /property %}
+
+  {% property name="accountId" type="string" %}
+    The [Account](/api/accounts) that the Notification was triggered for.
+  {% /property %}
+
+  {% property name="test" type="boolean" %}
+    Whether the Notification is for testing purposes.
+  {% /property %}
+
+  {% property name="createdAt" type="timestamp" %}
+    When the notification was created.
+  {% /property %}
+{% /properties %}
+
+## Notification Types
```

Comment:
@Ray2126 Change: Inline event payloads for each type to reduce indirection for readers and coupling to our activity models

Example: [Github](https://docs.github.com/en/webhooks/about-webhooks) - this is actually part of their guide, not their API reference

Reply:
I disagree, I think they should be coupled to our other models. This way, we can provide a consistent schema for the models that we expose to integrators.

Example: [Stripe](https://docs.stripe.com/api/events/types) - they link out to their other models for each Event type

Reply:
As discussed, we agree that notifications should reference the model to provide consistency